create_coda_files/docker-run.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME=maap-get_coda_files
+
+# Check that the correct number of arguments were provided.
+if [[ $# -lt 5 || $# -gt 6 ]]; then
+    echo "Usage: sh docker-run.sh <user> <json-input-path> <flow-name> <variable-name> <coda-output-path> <--prev-coda-input-path>"
```

Comment:
Optional arguments are conventionally come first.

Reply:
Is this in the python file as well or just in the bash script?

Reply:
And how does it affect how the script runs?

Reply:
Both.

`python script.py --optional x a b c` instead of `python script.py a b c --optional x`

Reply:
Optionals are a little painful in bash - here's an example of how to do it https://github.com/AfricasVoices/RapidProTools/blob/master/fetch_contacts/docker-run.sh

Reply:
Alternatively, you can make the prev-coda-input-path compulsory, but allow that directory to be empty, which is what REACH did.

Reply:
Lemme do that, looks easier.